package.json

```diff
@@ -39,6 +41,10 @@
     "test:e2e:prod": "npm run servebuild & npm run test:e2e; kill $!"
   },
   "dependencies": {
+    "expo": "~41.0.1",
+    "expo-status-bar": "~1.0.4",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz",
+    "react-native-web": "~0.13.12",
```

Comment:
Why downgrade?

Reply:
We ensure that the version of the Expo SDK and React Native community packages that you install are compatible with the app by running `expo install`.

[Ref](https://docs.expo.io/introduction/walkthrough/)

Reply:
Okay, thanks!